app/components/profile/picture_component.rb

```diff
@@ -26,4 +26,8 @@ def profile_picture_src
       'placeholder_profile_picture.png'
     end
   end
+
+  def open_to_work?
+    profile.job_search_status == 'open_to_work'
```

Comment:
shouldn't you be able to do `profile.open_to_work?` because it is an enum. Argueable, you can now just use this directly in the `erb` file because it is part of the model anyways.

Reply:
I thought so, but tried it earlier and it didn't work. I might've messed it up, tho, so will revisit tomorrow re-update if it doesn't complain